index.html

```diff
@@ -305,18 +305,32 @@ <h4>Ready?</h4>
         </section>
       </section>
       <section id="uartview" class="panel">
-        <input type="button" id="clearterm" value="Clear" />
-        <label for="baudrate">Baudrate: </label>
-        <select id="baudrate">
-          <option value="57600">57600</option>
-          <option value="38400">38400 </option>
-          <option value="19200">19200</option>
-          <option value="9600">9600</option>
-        </select>
-        <label for="uart-file">Send a file:</label>
-        <input type="file" id="uart-file" placeholder="Send a file through UART" />
-        <input type="button" id="uart-file-send" value="Send" />
-        <div id="sending">Sending...</div>
+        <div class="panel-toolbar">
+          <input type="button" id="clearterm" value="Clear" />
+          <label>
+            Baudrate:
+            <select id="baudrate">
+              <option value="57600">57600</option>
+              <option value="38400">38400 </option>
+              <option value="19200">19200</option>
+              <option value="9600">9600</option>
+            </select>
+          </label>
+          <label>
+            File:
+            <input type="file" id="uart-file" placeholder="Send a file through UART" />
+          </label>
+          <button id="uart-file-send">Send</button>
+          <input type="number" id="uart-char" value="21" style="width: 5ch" />
+          <input type="button" id="uart-char-send" value="Send Byte" />
+
+          <label>
+            Rows: <input type="number" value="80" style="width: 5ch;" id="uart-cols" />
+          </label>
+          <label>
+            Cols: <input type="number" value="24" style="width: 5ch;" id="uart-rows" />
```

Comment:
The label should enclose the input in HTML
```
<label>Cols:</label>
<input type="number" value="24" style="width: 5ch;" id="uart-rows" />
```

Also, `uart-rows` and `uart-cols` are swapped :smile:

Reply:
I swapped the Rows/Cols labels around - oops ;)